### PR TITLE
[MAIN-1886] Remove cves

### PR DESCRIPTION
**Pull request by Avi-Robusta**

### CODE REVIEW

**Comment by Avi-Robusta** [comment]
needed to remove gcloud-crc32c bc of vunerability, so i had to change how we install gcloud

File: Dockerfile [hunk 1]
```diff
@@ -31,13 +31,14 @@ RUN apt-get update \
...
 
-# Install Google Cloud SDK and Gcloud Auth Plugin
-RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list \
-    && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
-
-RUN apt-get update \
-    && apt-get install -y google-cloud-sdk google-cloud-sdk-gke-gcloud-auth-plugin \
-    && rm -rf /var/lib/apt/lists/*
+# Install Google cli so kubectl works w/ gke clusters
+RUN curl https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz > /tmp/google-cloud-sdk.tar.gz
```
Comment (Sheeproid) [inline Dockerfile:35]: I believe that /tmp/google-cloud-sdk.tar.gz is downloaded but never deleted. please delete after download to reduce image size.

File: Dockerfile [hunk 2]
```diff
@@ -31,13 +31,14 @@ RUN apt-get update \
...
+RUN mkdir -p /usr/local/gcloud \
+  && tar -C /usr/local/gcloud -xvf /tmp/google-cloud-sdk.tar.gz \
+  && /usr/local/gcloud/google-cloud-sdk/install.sh
+ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
```
Comment (Sheeproid) [inline Dockerfile:39]: I trust you've done relevant testing here

**Review by Sheeproid — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (Avi-Robusta) [reply Dockerfile:39]: ran it in docker, did gcloud auth and then connected to the cluster and saw data in kubectl

**Pull request merged**